polyfills/String/prototype/padEnd/config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"aliases": [
+		"es7"
+	],
+	"browsers": {
+		"android": "*",
+		"bb": "*",
+		"chrome": "<57",
+		"firefox": "<51",
```

Comment:
According to MDN, Firefox has had padEnd since version 48 -- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd#Browser_compatibility